COMPATIBILITY.md

```diff
@@ -28,6 +28,26 @@ You can add some labels to issue in jira:
             labels: [ "$version", "jenkins" ]])            
 ```
 
+##SearchIssuesStep
+
+Workflow custom pipeline step (see [step-api](https://github.com/jenkinsci/workflow-plugin/blob/master/step-api/README.md)) that allow to search by jql query directly from workflow.
```

Comment:
Now it's Pipeline. See the thread here: https://issues.jenkins-ci.org/browse/JENKINS-31153